semeio/jobs/overburden_timeshift/ots.py

```diff
@@ -49,10 +48,13 @@ def ots_load_params(input_file):
 
 
 def write_surface(vintage_pairs, ts, output_dir, type_str):
+    output_dir += type_str
+    if not os.path.isdir(output_dir):
+        os.mkdir(output_dir)
     for iv, vp in enumerate(vintage_pairs):
         d0 = vp[0].strftime("%Y_%m_%d")
         d1 = vp[1].strftime("%Y_%m_%d")
-        ts[iv].write(output_dir + type_str + "/ots" + "_" + d0 + "_" + d1 + ".irap")
+        ts[iv].to_file(os.path.join(output_dir, "ots" + "_" + d0 + "_" + d1 + ".irap"))
```

Comment:
Should the extension be the same for xtgeo surface?

Reply:
It is still `irap`, so I suppose so, but this implementation is failing for one of the tests, so might not be feasible to replace.